tests/core.py

```diff
@@ -1636,6 +1636,13 @@ def test_fetch_task_instance(self):
         response = self.app.get(url)
         self.assertIn("run_this_last", response.data.decode('utf-8'))
 
+    def test_missing_dag_request(self):
```

Comment:
Please add an opposite assert as well please (ie. with an existing DAG it shouldn't do this).